src/Symfony/Component/Translation/Bridge/Loco/composer.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "symfony/loco-translation",
+    "name": "symfony/loco-translation-provider",
```

Comment:
@fabpot we suffix the notifier with `notifier`, `-translation-provider` is more clear than `-translation`

Reply:
cc @welcoMattic

Reply:
`loco-translator`?

Reply:
I agree that adding `-translation-provider` to the provider name looks better.